deploy.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+
+# run this script as normal user w/o sudo from within the gin-repo root folder.
```

Comment:
`deploy` user doesn't have `sudo` rights. The script should actually be run as a user with `sudo` rights but it should be left to the script to use `sudo` where necessary.

It's not the best solution and perhaps it should be changed, but that's how it is for now. The main issue, if I remember correctly, is environment variables and paths, all of which can probably be handled more elegantly. Maybe it's best to rewrite the script to be run as `root`, not have it rely on the user's environment, and have it set ownerships where necessary.